spec.emu

```diff
@@ -70,7 +70,69 @@ contributors: Nicolò Ribaudo
         </table>
       </emu-table>
 
-      <emu-clause id="sec-module-namespace-exotic-objects-get-p-receiver" type="internal method" number="8">
+      <emu-clause id="sec-module-namespace-exotic-objects-getownproperty-p" type="internal method" number="5">
+        <h1>
+          [[GetOwnProperty]] (
+            _P_: a property key,
+          ): either a normal completion containing either a Property Descriptor or *undefined*, or a throw completion
+        </h1>
+        <dl class="header">
+          <dt>for</dt>
+          <dd>a module namespace exotic object _O_</dd>
+        </dl>
+        <emu-alg>
+          1. If <del>_P_ is a Symbol</del><ins>IsSymbolLikeNamespaceKey(_P_, _O_)</ins> is *true*, return OrdinaryGetOwnProperty(_O_, _P_).
```

Comment:
this should return `undefined` for "then", not something like `{ configurable: false, value: undefined }` for "then", is that correct?

Reply:
Yes, exactly like it does for symbols.